include/mlir/Analysis/AffineStructures.h

```diff
@@ -677,7 +677,7 @@ class FlatAffineConstraints {
   /// constraints, does a single scan, and updates in place.
   void removeTrivialRedundancy();
 
-  /// A more expensive check to detect redundant inequalities thatn
+  /// A more expensive check to detect redundant inequalities than
```

Comment:
I should revert this and leave it. This is because I cannot understand the original intention.